wait_for_local_port.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I would rather put this script in the tests/utils/ directory to avoid cluttering the root.